test/foundry/deploy/03_Erc20Enforcer.t.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.13;
+
+import {Test, Vm, console} from "forge-std/Test.sol";
+import {Constants} from "../utils/Constants.sol";
+import {HolographERC20} from "../../../src/enforcer/HolographERC20.sol";
+
+contract Erc20Enforcer is Test {
```

Comment:
suggestion: same here, we should add Test at the end

Reply:
delete file!